driver/controller.go

```diff
@@ -806,11 +806,33 @@ func (d *Driver) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsReques
 	return listResp, nil
 }
 
+// ControllerExpandVolume is called from the resizer to increase the volume size.
 func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	d.log.WithField("method", "controller_expand_volume").
-		Info("controller expand volume called")
+	ll := d.log.WithField("method", "controller_expand_volume")
+	ll.Info("controller expand volume called")
+	volID := req.GetVolumeId()
 
-	return nil, status.Error(codes.Unimplemented, "")
+	if len(volID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "volume ID missing in request")
+	}
+	ll.Info("extracting storage from requested capacity range")
+	resizeBytes, err := extractStorage(req.GetCapacityRange())
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, err.Error())
```

Comment:
We should probably mimic the pre-existing error handling of the `extractStorage` call and return a `codes.OutOfRange` code in order to [fulfill the CSI spec with regards to the expected error for _"Unsupported `capacity_range`"_](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerexpandvolume-errors).